$OpenBSD: patch-compat_Makefile_am,v 1.2 2019/11/16 18:28:19 ajacoutot Exp $

Trying to link a static archive with non-PIC objects to
plugins is not portable.

Index: compat/Makefile.am
--- compat/Makefile.am.orig
+++ compat/Makefile.am
@@ -1,7 +1,7 @@
-pkglib_LTLIBRARIES = libcompat.la
+noinst_LTLIBRARIES = libcompat.la
 libcompat_la_SOURCES = dummy.c
 libcompat_la_LIBADD = $(LTLIBOBJS)
-libcompat_la_LDFLAGS = -no-undefined -static
+libcompat_la_LDFLAGS = -no-undefined
 
 BUILT_SOURCES = dummy.c
 CLEANFILES = dummy.c
